Makefile

```diff
@@ -9,6 +10,11 @@ lint:
 		cargo clippy --all --all-targets --all-features --no-deps -- --deny warnings
```

Comment:
I didn't copy `--deny warnings` because it will stop the script. I think it's better to list all warnings at once. Another reason is that lint has expected errors.